src/test/java/cms/gov/madie/measure/services/TestCaseValidationServiceTest.java

```diff
@@ -400,4 +405,84 @@ public void testValidateTestCasesAsResourcesWithEntries() {
     assertThat(output.get(0).getHapiOperationOutcome(), is(notNullValue()));
     assertThat(output.get(0).getHapiOperationOutcome().getCode(), is(equalTo(200)));
   }
+
+  @Test
+  public void testValidateTestCaseAsynchronouslyForImport() {
+    Measure validingTestCaseMeasure =
+        measure.toBuilder()
+            .testCases(
+                List.of(
+                    testCase.toBuilder()
+                        .testCaseValidationStatus(TestCaseValidationStatus.PENDING)
```

Comment:
If this is meant to represent an active validation (the task is executing on a thread), then the expected status should be `VALIDATING`.